enterprise-suite/es-grafana/akka-exporter.json

```diff
@@ -1,13 +1,13 @@
 [  
   {"graphName":"Mailbox Size", "promQL":["max by (actor)(akka_actor_mailbox_size{workload=\"$workloadId\"})"]},
   {"graphName":"Mailbox Time", "promQL":["max by (actor)(akka_actor_mailbox_time_ns{workload=\"$workloadId\",quantile=\"0.99\"})"]},
-  {"graphName":"Actor Failures", "promQL":["sum by (actor) (akka_actor_actor_failure{workload=\"$workloadId\"})"]},
-  {"graphName":"Processed Messages", "promQL":["sum by (actor)(irate(akka_actor_processed_messages{workload=\"$workloadId\"}[1m]))"]},      
+  {"graphName":"Actor Failures per 15 Minutes", "promQL":["sum by (actor) (irate(akka_actor_actor_failure{workload=\"$workloadId\"}[15m]))"]},
+  {"graphName":"Processed Messages per Minute", "promQL":["sum by (actor)(irate(akka_actor_processed_messages{workload=\"$workloadId\"}[1m]))"]},      
```

Comment:
irate is "per second"

Reply:
again the 1m here just means compute the rate if there's a second value within 1 minute of the current value